protected/humhub/modules/space/models/Membership.php

```diff
@@ -170,24 +170,53 @@ public static function GetUserSpaces($userId = "", $cached = true)
         $spaces = Yii::$app->cache->get($cacheId);
         if ($spaces === false || !$cached) {
 
-            $orderSetting = Yii::$app->getModule('space')->settings->get('spaceOrder');
-            $orderBy = 'name ASC';
-            if ($orderSetting != 0) {
-                $orderBy = 'last_visit DESC';
-            }
-
-            $query = self::find()->joinWith('space')->orderBy($orderBy);
-            $query->where(['user_id' => $userId, 'space_membership.status' => self::STATUS_MEMBER]);
-
             $spaces = [];
-            foreach ($query->all() as $membership) {
+            foreach (static::getMembershipQuery($userId)->all() as $membership) {
                 $spaces[] = $membership->space;
             }
             Yii::$app->cache->set($cacheId, $spaces);
         }
         return $spaces;
     }
 
+    /**
+     * Returns a list of all spaces' ids of the given userId
+     *
+     * @param integer $userId
+     * @since 1.2.5
+     */
+    public static function GetUserSpaceIds($userId = "")
```

Comment:
By http://docs.humhub.org/dev-coding-standards.html you could rename to `getUserSpaceIds($userId = '')`

Reply:
done.